src/settings/JobProfiles/JobProfileDetails/JobProfileDetails.test.js

```diff
@@ -0,0 +1,170 @@
+      translationsProperties
+    );
+  };
+  it('should display job profile details', () => {
+    renderJobProfileDetails();
+
+    const dialog = screen.getByRole('dialog');
+
+    expect(dialog).toBeVisible();
+
+    const headings = within(dialog).getAllByRole('heading', { name: jobProfile.name });
+
+    headings.forEach(heading => expect(heading).toBeVisible());
+
+    const summary = within(dialog).getByRole('region', { name: /summary/i });
+
+    const labelsAndValues = [
+      'Record created: 12/4/2018 11:22 AM',
+      'Record last updated: 12/4/2018 1:28 PM',
+      commonTranslations.name,
+      jobProfile.name,
+      translations.description,
+      jobProfile.description,
+      translations.protocol,
+      '-',
+      translations.mappingProfile,
+      mappingProfile.name,
+    ];
+
+    labelsAndValues.forEach(el => expect(within(summary).getByText(el)).toBeVisible());
+  });
+  it('should display action buttons in the proper state', () => {
+    renderJobProfileDetails();
+    const actionButton = document.querySelector('[data-test-pane-header-actions-button]');
+
+    userEvent.click(actionButton);
+
+    const deleteButton = document.querySelector('[data-test-delete-profile-button]');
+    const duplicateButton = document.querySelector('[data-test-duplicate-profile-button]');
+    const editButton = document.querySelector('[data-test-edit-profile-button]');
+
+    expect(deleteButton).toBeDisabled();
+    expect(duplicateButton).toBeEnabled();
+    expect(editButton).toBeDisabled();
+  });
+  describe('rendering details without description for a job profile which is not already in use', () => {
+    const renderJobProfileWitoutDescription = () => {
+      renderWithIntl(
+        <SettingsComponentBuilder>
+          <JobProfileDetails
+            stripes={stripes}
+            jobProfile={{
+              ...jobProfile,
+              description: null,
+            }}
+            mappingProfile={mappingProfile}
+            isDefaultProfile={false}
+            isProfileUsed={false}
+            onCancel={noop}
+          />
+        </SettingsComponentBuilder>,
+        translationsProperties
+      );
+    };
+    it('should display no value in description', () => {
+      renderJobProfileWitoutDescription();
+      const description = document.querySelector('[data-test-job-profile-description]');
+
+      expect(within(description).getByText('-')).toBeVisible();
+    });
+    it('should display action buttons in the proper state', () => {
+      renderJobProfileWitoutDescription();
+      const actionButton = document.querySelector('[data-test-pane-header-actions-button]');
+
+      userEvent.click(actionButton);
+
+      const deleteButton = document.querySelector('[data-test-delete-profile-button]');
+      const duplicateButton = document.querySelector('[data-test-duplicate-profile-button]');
+      const editButton = document.querySelector('[data-test-edit-profile-button]');
```

Comment:
Let's get rid of translations when rendering, and use the translation keys to be able to use `getByRole`